### PR TITLE
executor: Add debug logs

### PR DESCRIPTION
**Pull request by zimulala**
Add transaction TS to log in debug level.

### CODE REVIEW

**Comment by zimulala** [comment]
PTAL @coocood @tiancaiamao @shenli @winkyao @hanfei1991

**Comment by shenli** [comment]
LGTM

File: executor/adapter.go [hunk 1]
```diff
@@ -90,7 +90,7 @@ func (a *recordSet) Next() (*ast.Row, error) {
 
 func (a *recordSet) Close() error {
 	err := a.executor.Close()
-	a.stmt.logSlowQuery(a.lastErr == nil)
+	a.stmt.logSlowQuery(nil, a.lastErr == nil)
```
Comment (coocood) [inline executor/adapter.go]: We can obtain the txnTS right after line 161 `buildExecutor`, and store it in recordSet

**Comment by coocood** [comment]
LGTM

**Pull request merged**